xbmc/Application.cpp

```diff
@@ -3501,6 +3501,8 @@ void CApplication::OnPlayBackEnded()
   CDarwinUtils::EnableOSScreenSaver(true);
 #endif
 
+  CServiceBroker::GetPVRManager().OnPlaybackEnded(m_itemCurrentFile);
```

Comment:
Absolutely, but not part/goal of this PR. This could be one of the next cleanup steps.